src/entities/offerings.ts

```diff
@@ -71,6 +76,56 @@ export interface Price {
   readonly formattedPrice: string;
 }
 
+/**
+ * Represents the price and duration information for a phase of the purchase option.
+ * @public
+ */
+export interface PurchaseOptionPhase {
```

Comment:
In purchases-android, there's a `PricingPhase`, which seems to hold very similar info. If they are indeed describing the same concept, should this be renamed to `PricingPhase` for consistency?

Reply:
Good point about all these! No reason to differentiate the name I think. Will rename those, thanks!